### PR TITLE
Add axe-core configuration injection

### PR DESCRIPTION
**Pull request by JediSange**
**Summary**
Currently the plugin does not allow for any configuration to be passed to the `axe-core` library. The [protractor-accessibility-plugin](https://github.com/angular/protractor-accessibility-plugin) project allows for that by passing on the `axe` object from the plugin config. This change mimics that as well as pulls in existing work from another open pull request #7.

**Further Reading**
- https://github.com/angular/protractor-accessibility-plugin/issues/14
- https://github.com/angular/protractor-accessibility-plugin/blob/master/index.js#L248
- https://github.com/dequelabs/axe-core/blob/develop/doc/API.md#api-name-axeconfigure

### CODE REVIEW

**Comment by JediSange** [comment]
@sachams since I can't assign the PR

File: index.js [hunk 1]
```diff
@@ -23,7 +23,7 @@ var AxeBuilder = require('axe-webdriverjs');
 // Global method, accessible from tests. 
 runAxeTest = function(testName, driver) {
   return new Promise((resolve, reject) => {
-    AxeBuilder(driver)
+    AxeBuilder(driver).options(this.config.axe)
```
Comment (etjung) [inline index.js]: Can we add this change to runAxeTestWithSelector a well?

**Comment by etjung** [comment]
@sachams Any possibility of getting this reviewed and possibly merged?

File: index.js (hunk 1 above)
Comment (JediSange) [reply index.js]: That's a good call. I can follow up on this. Unfortunately, it seems to be abandonware for the moment. We are maintaining our own branch for the time being and I'll add that to ours.

Comment (JediSange) [reply index.js]: Just added it. Let me know if that works for you. Feel free to PR into the repo we maintain at https://github.com/Snagajob/protractor-axe-report-plugin if you need more done.

**Pull request closed without merging**